python/lsst/ts/salobj/topics/read_topic.py

```diff
@@ -531,6 +550,8 @@ def get_oldest(self):
     async def next(self, *, flush, timeout=None):
         """Wait for a message, possibly returning the oldest queued message.
 
+        This reads and pops an item from the queue.
```

Comment:
but the item is still available from `get` (or `aget` with flush=False, yes?)
Is it correct to say,
"This reads and pops an item from the queue, but leaves the cache (used by `get` and `aget` unaltered)"

Reply:
Good point. Yes it does not affect the value returned by aget or get. I will add that information.

Reply:
Good question. I have reworded the text of all data reading methods to make it explicit what affect they have on other data reading methods. Please have another look.